bootstrap.d/x11-libs.yml

```diff
@@ -2027,3 +2027,28 @@ packages:
       - args: ['make', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: libxcvt
```

Comment:
Missing metadata, feel free to add it if you feel like maintaining this part of the Xorg stack. If you rather have me maintain it, let me know and I'll take it.
Also, you're missing the `labels: [aarch64]` thing, as it stands this build _will_ fail on xbbs for aarch64.